model/scheduled_task_test.go

```diff
@@ -73,3 +73,26 @@ func TestCancelTask(t *testing.T) {
 	time.Sleep(TASK_TIME + time.Second)
 	assert.EqualValues(t, 0, atomic.LoadInt32(executionCount))
 }
+
+func TestCreateRecurringTaskFromNextIntervalTime(t *testing.T) {
+	TASK_NAME := "Test Recurring Task starting from next interval time"
+	TASK_TIME := time.Second * 5
+
+	var executionTime time.Time
+	testFunc := func() {
+		executionTime = time.Now()
+	}
```

Comment:
This is racy because the ticker runs in a goroutine. Wrap this with a mutex, and always run your test with `-race` to verify that there are no races.